google/cloud/speech/cloud_speech.yaml

```diff
@@ -11,17 +11,18 @@ documentation:
     Google Cloud Speech API.
 
 apis:
-- name: google.cloud.speech.v1.Speech
+- name: google.cloud.speech.v1beta1.Speech
+- name: google.longrunning.Operations
```

Comment:
I recommend against commenting it out in the proto. That makes integration from the source of truth harder. I also recommend against commenting it out in config, in the transitional case where some languages support it but not others. I think the best approach is through ignoring the methods in code, so each language is capable of including them when they support LRO methods.